### DISCUSSION TITLE
Getting a call to the modules once treesitter plugin is initialized

### DISCUSSION THREAD

**Discussion by Badhi** [nvim-treesitter/nvim-treesitter#4465] [Q&A] [answered]
I have a module that gets plugged into this treesitter plugin. At initialization of my plugin I want to get the `config` table data from the `treesitter.config`. But since the initialization order is not strict, my plugin may get initialized before the treesitter plugin is initialized. Is there a call that a module can receive after the treesitter plugin is initialized? Or is there a better way to ensure that the modules of the tresitter are initialized after the treesitter is initialized.

**Comment by clason** [comment, marked as answer]
The better approach is to turn your plugin into a stand-alone; see https://github.com/nvim-treesitter/nvim-treesitter-context for an example.
